web/templates/note/form.html.eex

```diff
@@ -0,0 +1,13 @@
+<%= form_for @changeset, @action, fn f -> %>
+  <div>
+      <%= label f, :title , "Title" %>
+      <%= text_input f, :title %>
+  </div>
+  <div>
+    <%= label f, :description , "Description" %>
+    <%= text_input f, :description %>
+  </div>
+  <%= submit submit_text(@action) %>
```

Comment:
Instead of this extra layer of indirection, why not just pass the desired text in?